diff --git a/src/commands/unlock_secret_mode.c b/src/commands/unlock_secret_mode.c
index 020d8c2..654b72e 100644
--- a/src/commands/unlock_secret_mode.c
+++ b/src/commands/unlock_secret_mode.c
@@ -14,7 +14,6 @@ void unlock_secret_mode()
 {
     if (IS_ADMIN) {
         printf("Secret mode unlocked! Welcome, admin.\n");
-        printf("{ADMIN4242}\n");
     } else {
         printf("Access denied. You do not have the required privileges.\n");
     }
